apps/demos/testing/widgets/chat/Customization.test.js

```diff
@@ -9,12 +9,12 @@ fixture('Chat.Customization')
     ctx.initialWindowSize = [900, 800];
   });
 
-runManualTest('Chat', 'Customization', ['jQuery'], (test) => {
+runManualTest('Chat', 'Customization', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
   test('Customization', async (t) => {
     const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
     await t
-      .click('#day-headers-format')
```

Comment:
"You use IDs for checkboxes in the jQuery demo only. To make it work for all approaches, we should unify the method of selecting the correct checkbox, e.g., Selector('.dx-checkbox').nth(2)

![Uploading image.png…]()

Reply:
[37c8016](https://github.com/DevExpress/DevExtreme/pull/28482/commits/37c8016387efd0b5991f5f9cb043fba1c686bd51)